doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,188 @@
 [<< Back](../)
 
 # 1. Introduction
+<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
+
+* [1.1 Introduction](#1.1)
+  * [1.1.1 Terminology](#1.1)
+  * [1.1.2 Relation to other communities](#1.1)
+* [1.2 Scope](#1.2)
+* [1.3 Principles and Guidelines](#1.3)
+  * [1.4.1 Overarching Objectives and Goals](#1.4.1)
+* [1.4 Best Practices](#1.4)
+* [1.5 Verification methodologies](#1.5)
+*	[1.6 Assumptions & Dependencies](#1.6)
+*	[1.7 Results Collation & Presentation](#1.7)
+* [1.9 Resources & References](#1.9)
+
+## Executive Summary
+
+The Reference Conformance for the Kubernetes-based workstream (RC2) was established to ensure implementations of the CNTT Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet functional and performance requirements specified in RA2 and the CNTT Reference Model (RM). Cloud infrastructure and cloud native network function (CNF) verification and validation will be utilised to evaluate **Conformance** (i.e. adherence) to the RA2 and RM specifications. Conformance scope includes:
+
+ - Test criteria, with requirements traceability, to validate that the cloud infrastructure implementation meets the expected capabilities specified in RA-2 and that the CNFs consume compliant cloud infrastructure resources
+ - Verify, with requirements traceability, that the installation cookbooks (manifests) are in conformance with the RA-2 specifications (for example, software versions, plugins, and configurations).
```

Comment:
validation of "installation cookbooks" should refer to RI2. For a given vendor implementation, we should not refer to the installation cookbooks, but just to the system under test - as described in the first bullet. 
```suggestion
 - Verify, with requirements traceability, that the installation cookbooks (manifests) of RI-2 are in conformance with the RA-2 specifications (for example, software versions, plugins, and configurations).
```